Makefile.am

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+SUBDIRS = src
+
+EXTRA_DIST =
+EXTRA_DIST += autogen.sh
+EXTRA_DIST += include
+EXTRA_DIST += debian
+EXTRA_DIST += nccl-rdma-sharp-plugins.spec
+EXTRA_DIST += nccl-rdma-sharp-plugins.spec.in
```

Comment:
why .in file in EXTRA? it should appear only in AC_OUTPUT as template and will be distributed automatically

Reply:
fixed in new PR